docs/en/12-taos-sql/22-meta.md

```diff
@@ -102,7 +110,11 @@ Provides information about user-created databases. Similar to SHOW DATABASES.
 | 24  | wal_retention_period | INT              | WAL retention period                                   |
 | 25  |  wal_retention_size  | INT              | Maximum WAL size                                   |
 | 26  |   wal_roll_period    | INT              | WAL rotation period                                 |
-| 27  |   wal_segment_size   | WAL file size |
+| 27  |   wal_segment_size   | BIGINT | WAL file size |
+| 28  |   stt_trigger   | SMALLINT | The number of files placed on the disk that trigger file merging |
+| 29  |   table_prefix   | SMALLINT | The table name is used for the prefix length that needs to be ignored when calculating the consistent HASH |
+| 30  |   table_suffix   | SMALLINT | The table name is used for the suffix length that needs to be ignored when calculating the consistent HASH |
```

Comment:
The suffix length in the table name that's ignored when distributing table to vnode based on table name